2.1-segwit-version-1.ipynb

```diff
@@ -284,7 +294,130 @@
     "privkey1, pubkey1 = generate_key_pair()\n",
     "privkey2, pubkey2 = generate_key_pair()\n",
     "\n",
-    "# Generate a 2-of-2 aggregate MuSig key\n",
+    "# Create the spending script\n",
+    "multisig_script = CScript([CScriptOp(OP_2), pubkey1.get_bytes(), pubkey2.get_bytes(), CScriptOp(OP_2), CScriptOp(OP_CHECKMULTISIG)])\n",
+    "\n",
+    "# Hash the spending script\n",
+    "script_hash = sha256(multisig_script)\n",
+    "\n",
+    "# Generate the address\n",
+    "version = 0\n",
+    "address = program_to_witness(version, script_hash)\n",
+    "print(\"bech32 address is {}\".format(address))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example 2.1.8: Start a Bitcoind node and send funds to the segwit v0 address\n",
+    "\n",
+    "We'll use the `generate_and_send_coins()` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test = util.TestWrapper()\n",
+    "test.setup()\n",
+    "node = test.nodes[0]\n",
+    "\n",
+    "# Generate coins and create an output\n",
+    "tx = node.generate_and_send_coins(address)\n",
+    "print(\"Transaction {}, output 0\\nsent to {}\\n\".format(tx.hash, address))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Example 2.1.9 : Construct CTransaction and populate fields"
```

Comment:
Suggest more detail in this title or in a paragraph below. We are signing and testing for acceptance too.

Reply:
Agree. I've updated the title and added more detail.